spartan/scripts/deploy_kind.sh

```diff
@@ -60,13 +61,38 @@ fi
 # and are used to generate the genesis.json file.
 # We need to read these values and pass them into the eth devnet create.sh script
 # so that it can generate the genesis.json and config.yaml file with the correct values.
-./generate_devnet_config.sh "$values_file"
+if [ "$sepolia_deployment" != "true" ]; then
+  echo "Generating devnet config..."
+  ./generate_devnet_config.sh "$values_file"
+fi
 
 # Install the Helm chart
+echo "Cleaning up any existing Helm releases..."
+helm uninstall spartan -n "$namespace" 2>/dev/null || true
+kubectl delete clusterrole spartan-aztec-network-node 2>/dev/null || true
+kubectl delete clusterrolebinding spartan-aztec-network-node 2>/dev/null || true
+
+helm_set_args=(
+  --set images.aztec.image="aztecprotocol/aztec:$aztec_docker_tag"
+)
+
+# If this is a sepolia run, we need to write some values
+if [ "$sepolia_deployment" = "true" ]; then
+  helm_set_args+=(
+    --set ethereum.externalHost="$EXTERNAL_ETHEREUM_HOST"
+    --set ethereum.beacon.externalHost="$EXTERNAL_ETHEREUM_CONSENSUS_HOST"
+    --set ethereum.beacon.apiKey="$EXTERNAL_ETHEREUM_CONSENSUS_HOST_API_KEY"
+    --set ethereum.beacon.apiKeyHeader="$EXTERNAL_ETHEREUM_CONSENSUS_HOST_API_KEY_HEADER"
+    --set aztec.l1DeploymentMnemonic="$L1_ACCOUNTS_MNEMONIC"
+    --set ethereum.deployL1ContractsPrivateKey="$L1_DEPLOYMENT_PRIVATE_KEY"
+    --values "../aztec-network/values/$values_file"
```

Comment:
I wouldn't lump values file here

Reply:
curious why not? you think better to extract each value specifically or..?

Reply:
Also surely it should be unconditionally present

Reply:
Ah right 😬 
Yeah I'd think so but also thought there's a reason it wasn't present already below? will add

Reply:
PR showed you removed it

Reply:
oh my 🫣